README.md

```diff
@@ -1,12 +1,14 @@
 # Gallactic [![Build Status](https://api.travis-ci.org/gallactic/gallactic.svg?branch=master)](https://travis-ci.org/gallactic/gallactic)
+
 *Gallactic blockchain with [SputnikVM](https://github.com/gallactic/sputnikvm) and [Tendermint](https://github.com/tendermint/tendermint) consensus engine*
 
 ## Compiling the code
+
 You need to make sure you have install [Go](https://golang.org/) (version 1.10.1 or higher) and [rust](https://www.rust-lang.org). After installing them, you can follow these steps to compile and build the gallactic project:
 
-```
+```bash
 mkdir -p $GOPATH/src/github.com/gallactic/gallactic
-cd $GOPATH/src/github.com/gallactic
+cd $GOPATH/src/github.com/gallactic/gallactic
 git clone https://github.com/gallactic/gallactic.git .
 make
```

Comment:
after git clone 
`cd gallactic` should be included else it cannot find the make file .

Reply:
`cd $GOPATH/src/github.com/gallactic/gallactic` in this approach we are in the `galalctic/galalctic` folder. we clone the project in same folder. Look at the dot at the end of clone command. It means clone here.

Reply:
okay great thanks